content/updates/2021-04/2-Mass-delete-records.md

```diff
@@ -0,0 +1,19 @@
+---
+pim_announcement_img: ./img/reference-entity-records-mass-delete.png
+pim_announcement_alt_img: "Reference entity records mass delete screen"
+pim_announcement_audience:
+- EE
+---
+
+# Mass delete your reference entity records
+::: meta-data type="New" features="Productivity" available="mid-April" in="EE" link-to-doc="../articles/enrich-your-reference-entity-records.html#delete-a-selection-of-records"
+
+Have you ever needed to clean up your reference entity records? If yes, you can now do this very quickly within just a few clicks directly from your reference entity record grid. Indeed, you can now delete several records from the same reference entity at the same time. No doubt this will save you a lot of time!
```

Comment:
```suggestion
Do you ever need to clean up your reference entity records? It's super easy now, you can do it very quickly in a couple of clicks directly from your reference entity record grid. Indeed, you can now delete several records from the same reference entity at the same time. No doubt this will save you a lot of time!
```